shared_model/validators/default_validator.hpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_DEFAULT_VALIDATOR_HPP
+#define IROHA_DEFAULT_VALIDATOR_HPP
+
+#include "validators/commands_validator.hpp"
+
+namespace shared_model {
+  namespace validation {
+    using DefaultValidator = CommandsValidator;
```

Comment:
1. Don't really sure that it's meaningful, it's only used in tx builder. So maybe remove and just use CommandsValidator?
2. It's a **command** validator, it's not applicable to query, name is wrong

Reply:
The idea is that we can create any validators and define one of them as default one. We will create another PR for stateless validation for queries

Reply:
It's just weird